.github/workflows-source/fi-daily.yml

```diff
@@ -0,0 +1,82 @@
+name: FI Daily
```

Comment:
Can you name it "Schedule Daily FI"? And also name the workflow file `schedule-daily-fi.yml`. This makes logical grouping with current schedules and helps with readability.